resource_customizations/cdi.kubevirt.io/DataVolume/health.lua

```diff
@@ -0,0 +1,14 @@
+hs = { status="Progressing", message="No status available"}
+if obj.status ~= nil then
+  if obj.status.phase ~= nil then
+    hs.message = obj.status.phase
+    if hs.message == "Succeeded" then
+      hs.status = "Healthy"
+    elseif hs.message == "Failed" or hs.message == "Unknown" then
+      hs.status = "Degraded"
+    elseif hs.message == "Paused" then
+      hs.status = "Suspended"
+    end
```

Comment:
In addition to the phase, it may be worthwhile to look at the `Running` condition, and report `Progressing` only if the condition is true. For example, a DataVolume which points to a non-existing source, will be stuck forever in an `ImportInProgress` phase, and its importer pod will continuously crash-loop. Most of the time, the `Running` condition will be set to false (except, momentarily when the importer pod is relaunched, but due to the exponential backoff it'll happen less and less often).